docs/samples/v1beta1/torchserve/v1/bloom/README.md

```diff
@@ -0,0 +1,183 @@
+# TorchServe example with Huggingface BLOOM model
+In this example we will show how to serve [Huggingface Transformers with TorchServe](https://github.com/pytorch/serve/tree/master/examples/Huggingface_Transformers)
```

Comment:
Do you mean by `https://github.com/pytorch/serve/tree/master/examples/Huggingface_Largemodels` ?